packages/union-component/src/components/DonationFundDropdown.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { SelectHTMLAttributes, useEffect, useState } from 'react';
+import DonationDropdown from './DonationDropdown';
+import { getMembershipFunds, IFund } from './utils';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
```

Comment:
Can you also update `packages/union-component/src/components/DonationCountryDropdown.tsx` definition? it seems we are using interface on everything but that one and for a second I felt this one was _"wrong"_ _(in terms of being different)_

Reply:
well, it's fine that DonationCountryDropdown uses a type over an interface, I need an interface here since `SelectHTMLAttributes<HTMLSelectElement>` is an interface an I needed to extend it